packages/bitcore-wallet-service/src/config.ts

```diff
@@ -456,6 +456,7 @@ const Config = (): any => {
     //   },
     //   v2: {
     //     secret: 'changelly_secret_v2',
+    //     STABLECOIN_KEY: 'changelly_secret_stablecoin_v2'
```

Comment:
It'd be nice to have some naming & case consistency here. Perhaps `secret_stablecoin` or `secret_alt` would be better.